cromwell-compose-staging.yml

```diff
@@ -1,10 +1,10 @@
 version: '2.1'
 services:
-  ui:
+  jmui:
```

Comment:
@samanehsan @rexwangcc  IIRC one or more of these compose files are unused. Which ones can I delete?

Reply:
Sorry for this late reply, I think it is ok to remove the `cromwell-compose-staging.yml`, `environment.cromwell.staging.ts` and its registry in `.angular-cli.json`, since they are identical to the dev ones, except the names.

Reply:
Cool thanks, I will remove that in a follow-up PR.